ui/src/kapacitor/utils/ActiveKapacitorFromSources.ts

```diff
@@ -0,0 +1,18 @@
+import {Source, Kapacitor} from 'src/types'
+
+const ActiveKapacitorFromSources = (
+  source: Source,
+  sources: Source[]
+): Kapacitor => {
+  if (!source || !sources) {
+    return null
+  }
+  const ActiveSource = sources.find(s => s.id === source.id)
+  if (!ActiveSource || !ActiveSource.kapacitors) {
+    return null
+  }
+  const activeKapacitor = ActiveSource.kapacitors.find(k => k.active)
```

Comment:
just a tiny nit, can we just return the result of find and not assign a variable.